master/getting-started/kubernetes/installation/hosted/index.md

```diff
@@ -110,3 +110,55 @@ be filled in automatically by the `calico/cni` container:
 | `__ETCD_KEY_FILE__`                   | The path to the etcd key file installed to the host, empty if no key present.
 | `__ETCD_CERT_FILE__`                  | The path to the etcd cert file installed to the host, empty if no cert present.
 | `__ETCD_CA_CERT_FILE__`               | The path to the etcd CA file installed to the host, empty if no CA present.
+
+## Using calicoctl with a self-hosted installation
+
+> **NOTE** 
+>
+> The following information applies to the etcdv2 backend only. See the [etcdless hosted install](k8s-backend/)
+for how to run calicoctl in an etcdless installation.
+
+Using calicoctl is no different on a self-hosted installation.  However, the manifests above do not
+install calicoctl.
+
+You can install calicoctl by [downloading the appropriate release]({{site.baseurl}}/{{page.version}}/releases) to any 
+machine with access to your etcd cluster.
+
```

Comment:
Worth mentioning that you need to specify ETCD_ENDPOINTS when you run it - or else you end up maybe writing to k8s's ETCD instead of calico's.  Does the hosted install use a predictable IP for ETCD_ENDPOINTS?